.travis.yml

```diff
@@ -13,13 +13,14 @@ env:
   matrix:
     - IMAGE=base-notebook
     - IMAGE=pangeo-notebook
+    - IMAGE-pangeo-ocean
   global:
     - DOCKER_ID=pangeo
     - CALVER="$( date '+%Y.%m.%d' )"
 
 install:
   # pinned r2d version to fix conda install problems
-  - pip install git+git://github.com/jupyter/repo2docker.git@9766c9545540fb3e461c6ea92b054dbb3cb90184
+  - pip install git+git://github.com/jupyter/repo2docker.git@8dfd400c779763d0051975454050f22c1f19daf0
```

Comment:
@jhamman - in order to get my Julia env to work, I need a more recent repo2docker commit. Will this cause problems for the other images?